addons/stock/models/stock_move.py

```diff
@@ -1657,6 +1640,7 @@ def _action_assign(self, force_qty=False):
                 missing_reserved_uom_quantity = force_qty
             missing_reserved_uom_quantity -= reserved_availability[move]
             missing_reserved_quantity = move.product_uom._compute_quantity(missing_reserved_uom_quantity, move.product_id.uom_id, rounding_method='HALF-UP')
+            quant_ids = quants_by_product[move.product_id.id]
```

Comment:
quants = 
We use recordset with s at the end when it's a record set
and _ids when it's a list of ids

Reply:
We're not very consistent with that naming standard. I've seen it used both for recordsets where we expect more than 1 record and and when it's a list of ids. That being said, if we want to be consistent in this PR then you missed replacing another quant_ids = quants_by_product in move.line.unlink